apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx

```diff
@@ -21,12 +21,18 @@ export const enum PortalStatus {
 	void = "VOID",
 }
 
-async function Portal() {
+async function Portal({
+	searchParams,
+}: {
+	searchParams?: {
+		return_to?: string;
+	};
+}) {
```

Comment:
Thought: I don't think there's any situation where a user will be sent to `/portal` with a `return_to` parameter?